packages/framework/fluid-llm/.env

```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY="replaceMe"
```

Comment:
fyi--I know this is a draft, but in general version controlling the .env file is a bit dangerous as people may accidentally stage/commit it and not correctly rewrite history before pushing creds to somewhere public.

An alternative pattern I've seen which is basically as lightweight is to version control a file named `.env.sample` or something like that, which would have contents resembling this, and .gitignore `.env`. Then README instructions for setup have users copy the .sample file and populate creds locally.